### DISCUSSION TITLE
why bootstrap use sass instead of stylus

### DISCUSSION THREAD

**Discussion by abdo-spices** [twbs/bootstrap#36835] [Q&A] [answered]
why bootstrap use sass instead of stylus

**Comment by mdo** [comment, marked as answer]
We've used Sass since the migration from Less in v4. Stylus never seemed as widely used as Sass, and that remains true today as well. It's never really been on our radar.

**Reply by abdo-spices** [reply]
but stylus has a lot of features that are not found in sass 

and why bootstrap did not make ther self css preprosecor

**Reply by mdo** [reply]
Breathe of features isn't everything. Sass is more popular by a long shot. And we're not really interested in making our own at this time.
